Library/Homebrew/keg_relocate.rb

```diff
@@ -177,41 +177,36 @@ def recursive_fgrep_args
 
   def egrep_args
     grep_bin = "grep"
-    grep_args = recursive_fgrep_args
-    grep_args += "Pa"
+    grep_args = ["--files-with-matches",
+                 "--perl-regexp",
+                 "--binary-files=text"]
```

Comment:
```suggestion
    grep_args = [
      "--files-with-matches",
      "--perl-regexp",
      "--binary-files=text",
    ]
```

just if we decide to add/remove this later to make the diff nicer.